test/ffi/booster.jl

```diff
@@ -163,8 +163,32 @@ end
 
 @testset "LGBM_BoosterRollbackOneIter" begin
 
-    # I don't know how to test this, needs thought
-    @test_broken false
+    # Arrange
+    mymat = randn(10000, 2)
+    labels = randn(10000)
+    dataset = LightGBM.LGBM_DatasetCreateFromMat(mymat, verbosity)    
+    LightGBM.LGBM_DatasetSetField(dataset, "label", labels)
+    booster = LightGBM.LGBM_BoosterCreate(dataset, verbosity)
+
+    # update learning 20 times
+    for _ in [1:20;]
+        finished = LightGBM.LGBM_BoosterUpdateOneIter(booster)
+    end
+
+    # Act and Assert
+    for n in reverse([1:20;])
+        @test LightGBM.LGBM_BoosterGetCurrentIteration(booster) == n
+
+        model_string = LightGBM.LGBM_BoosterSaveModelToString(booster)
+        tree_sizes_in_string = match(r"\ntree_sizes=(.*)\n", model_string)[1]
```

Comment:
Thats good just put a comment to explain it

Reply:
Done :)